### PR TITLE
Allow creation and deletion of auth zones

### PR DESCRIPTION
**Pull request by alanplatt**
This PR adds all functionality needed to allow the creation/deletions of auth zones by terraform. My intention is to add support for this to the official provider when this PR is merged.

### CODE REVIEW

**Comment by alanplatt** [comment]
@somashekhar Hi, sorry to bother you but I see you've recently merged a PR. Is it possible to get this merged? I'm unclear if anything else is required or what the usual process is

**Comment by alanplatt** [comment]
@yuewko

File: object_manager.go [hunk 1]
```diff
@@ -752,6 +753,39 @@ func (objMgr *ObjectManager) GetGridInfo() ([]Grid, error) {
 	return res, err
 }
 
+// CreateZoneAuth creates zones and subs by passing fqdn
+func (objMgr *ObjectManager) CreateZoneAuth(fqdn string, ea EA) (*ZoneAuth, error) {
```
Comment (chinmayb) [inline object_manager.go:757]: is this method not needed to be part of interface `IBObjectManager` ?

Comment (alanplatt) [reply object_manager.go:759]: IBObjectManager updated

**Review by chinmayb — APPROVED** [review]

**Review by chinmayb — APPROVED** [review]

**Pull request merged**